app/components/ButtonGroup/ButtonGroup.js

```diff
@@ -9,25 +9,26 @@ export default function ButtonGroup({ buttons, }) {
   return (
     <div className="buttonGroup">
       {
-        buttons.map((button, index) => {
-          const {
+        buttons.map((
+          {
             text,
             classes,
             disabled,
             onClick,
-          } = button;
+          },
+          index,
+        ) => {
           const buttonProps = {
             type: 'button',
             key: `button${index}`,
             className: classes,
             style,
             onClick,
             disabled,
+            children: text,
           };
           return (
-            <button { ...buttonProps }>
-              {text}
-            </button>
+            <button { ...buttonProps } />
```

Comment:
Now that it's a single line expression, drop the parents: `return <button {...buttonProps} />`

Reply:
done.

now, this section looks not very clear to my eyes at the moment. I mean, this structure of a `div` containing a relatively long `map()` inside of it. 

I was thinking about taking out the function inside the map and putting it prior to the component's `return`.

you'll see it in the next commit. let me know what you think